lib/generators/manageiq/plugin/templates/bin/setup

```diff
@@ -4,8 +4,13 @@ require 'pathname'
 gem_root = Pathname.new(__dir__).join("..")
 
 unless gem_root.join("spec/manageiq").exist?
-  puts "== Cloning manageiq sample app =="
-  system "git clone https://github.com/ManageIQ/manageiq.git --branch master --depth 1 spec/manageiq"
+  if ENV.key?("MANAGEIQ_REPO")
```

Comment:
As discussed in https://github.com/ManageIQ/manageiq-providers-amazon/pull/570 , let's  give priority to the ENV var, else do what was there before, so...

```ruby
if ENV.key?("MANAGEIQ_REPO")
  # ln -sf
elsif !gem_root.join("spec/manageiq").exist?
  # git clone
end
```